aspnetcore/blazor/supported-platforms.md

```diff
@@ -0,0 +1,27 @@
+---
+title: ASP.NET Core Blazor supported platforms
+author: guardrex
+description: Learn about the supported platforms for ASP.NET Core Blazor.
+monikerRange: '>= aspnetcore-3.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 05/21/2019
+uid: blazor/supported-platforms
+---
+# ASP.NET Core Blazor supported platforms
+
+By [Luke Latham](https://github.com/guardrex)
+
+## Server system requirements
+
+[Blazor server-side](xref:blazor/hosting-models#server-side) supports any server platform that ASP.NET Core supports.
+
+## Browser requirements
```

Comment:
This browser matrix is actually specific to server-side Blazor.

For client-side Blazor we don't support IE at all, because it doesn't support WebAssembly.